src/styles/colors.ts

```diff
@@ -288,6 +288,9 @@ const getColorsByTheme = (darkMode?: boolean) => {
     whiteTransparent: buildRgba(base.white, 0.8), // '255, 255, 255'
   };
 
+  const statusColor = buildRgba(base.blueGreyDark, 0.7);
```

Comment:
check the gradients, it's all with the same thing. but there are even more things like `blueGreyDark60`, i guess we just got too lazy to define colors in the theme and keep using these alpha things for changing the "feel" of the color instead of following the design system. in my opinion we should get rid of inline alphas since it's impossible to be consistent with that.

Reply:
this is not increasing consistency. I don't think, we got too lazy, this is the same color with alpha applied. 
Gradients are different -they are not defining new colors

Reply:
I don't think that we should introduce wrong parttern to the code we basically refactor. Making fast components is not just about migration from styled components to raw view, but also refactoring components in general.

Reply:
I added an explanation of my reasoning here: 
https://rainbowhaus.slack.com/archives/C03B2U2Q4SG/p1657110228690219
If you don't agree, please, let's continue the discussion there